adadamp/_dist.py

```diff
@@ -311,3 +318,18 @@ def _get_gradients(
         grads = [grad_fn(idx=idx) for idx in worker_idxs]
         out = dask.compute(*grads)
         return out
+
+class DaskBaseDamper2(DaskBaseDamper):
+    
+    def score(self, X, y):
+        loss = super().score(X, y)
+        
+        correct = 0
+        total = 0
+        for Xi, yi in zip(X, y):
+            out = self.module_(Xi)
```

Comment:
I am also having some trouble here. I am not sure why, but I am unable to access the parent instance variables. It keeps telling me that `self.module_` is not a variable. This happens if i do `self.module_.forward(X)` aswell.

Reply:
Hm... that's strange. I suspect `initialize` hasn't been called yet; that's where `self.module_` is declared. https://github.com/stsievert/adadamp/blob/4302e836c35e900245ca4bdd29611866ff96613e/adadamp/_dist.py#L136-L144